task/buildah/0.1/buildah.yaml

```diff
@@ -146,6 +146,7 @@ spec:
       if [ -n "${PREFETCH_INPUT}" ]; then
         mv cachi2 /tmp/
         chmod -R go+rwX /tmp/cachi2
+        cp /tmp/cachi2/output/bom.json ./sbom-cachi2.json
```

Comment:
Which directory is `.`? Are we putting the SBOM in the root of the user's repo?

The user could accidentally `COPY . /foo` the SBOM into their image, can we avoid that?

Reply:
`.` is the [workspace](https://github.com/brunoapimentel/build-definitions/blob/merge-cachi2-sbom/task/buildah/0.1/buildah.yaml#L251) that gets mounted in the image (same one that got the cloned sources).

Note that this workspace is not added to the final image, which is already built at this point. This is [how](https://github.com/brunoapimentel/build-definitions/blob/merge-cachi2-sbom/task/buildah/0.1/buildah.yaml#L283) we inject the SBOMs to the image (to the usual `/root/buildinfo/content_manifests`)

The SBOM is also being [pushed](https://github.com/brunoapimentel/build-definitions/blob/merge-cachi2-sbom/task/buildah/0.1/buildah.yaml#L303-L313) to the registry.

Reply:
This part is before the build, `buildah build` happens at the end of this same script (though the build command is hidden in lots of `unshare` dark magic)

Reply:
Oh, good catch, I confused this with the `merge-cachi2-sbom` step.

I believe the `source` workspace is the only way to persist the content between steps as of now. I can try to make this copy after the build is done, it should solve the issue.